homeassistant/components/enphase_envoy/manifest.json

```diff
@@ -2,7 +2,7 @@
   "domain": "enphase_envoy",
   "name": "Enphase Envoy",
   "documentation": "https://www.home-assistant.io/integrations/enphase_envoy",
-  "requirements": ["envoy_reader==0.17.0"],
+  "requirements": ["envoy_reader==0.17.0rc0"],
```

Comment:
At first glance it looks like that according to PEP-440 is `0.17.0rc0` a [pre-release](https://www.python.org/dev/peps/pep-0440/#pre-releases) that would become `0.17.0`.

The latest release on PyPI is [0.17.2 ](https://pypi.org/project/envoy-reader/). It feels like that the changes should be made public as 0.17.3 to avoid confusion.

Reply:
I changed the version number from `0.17.0rc0` to `0.17.3`.

I agree the versioning scheme isn't the greatest.  I'll look into the versioning standard you mentioned.  Thanks! 😄